ferretdb/ferretdb.go

```diff
@@ -118,18 +121,20 @@ func New(config *Config) (*FerretDB, error) {
 
 	metrics := connmetrics.NewListenerMetrics()
 
-	log := config.Logger
+	log := config.SLogger
 	if log == nil {
 		log = getGlobalLogger()
 	} else {
-		log = logging.WithHooks(log)
+		log = logging.WrapLogger(log)
```

Comment:
Ok, but why? We should just use the passed logger. It is the caller's responsibility to pass the logger (and handler) configured to their desires

Reply:
If the logger isn't wrapped, `getLog` will be empty, and yes we can make that as the caller's responsibility. I'll make a comment about it on `Config.SLogger`.